Sources/Purchasing/TrialOrIntroPriceEligibilityChecker.swift

```diff
@@ -135,6 +135,20 @@ extension TrialOrIntroPriceEligibilityCheckerType {
 
 }
 
+/// Default overload implementation that exposes an `async` API.
+@available(iOS 13.0, tvOS 13.0, watchOS 6.2, macOS 10.15, *)
+extension TrialOrIntroPriceEligibilityCheckerType {
+
+    func checkEligibility(productIdentifiers: [String]) async -> [String: IntroEligibility] {
```

Comment:
Hmm, this seems unused other than in tests?

Reply:
it's actually [public API](https://github.com/RevenueCat/purchases-ios/blob/4.14.0/Sources/Purchasing/Purchases/Purchases.swift#L731).

Reply:
But that's the completion version, not this one with async right?

Reply:
Oooh good catch! I moved this to just the test file.